package.json

```diff
@@ -39,7 +50,11 @@
         "!dist/test/",
         "!dist/bin/",
         "!dist/package.json",
-        "!dist/tsconfig.tsbuildinfo"
+        "!dist/tsconfig.tsbuildinfo",
+        "!dist/esm/test/",
+        "!dist/esm/bin/",
+        "!dist/esm/package.json",
```

Comment:
🤔

Reply:
well, I just copied what was there for the cjs currently, did not really check if these files are generated